simple-mbed-cloud-client.h

```diff
@@ -51,11 +52,10 @@ class SimpleMbedCloudClient {
     MbedCloudClientResource* create_resource(const char *path, const char *name);
     void on_registered(Callback<void(const ConnectorClientEndpointInfo*)> cb);
     void on_unregistered(Callback<void()> cb);
-    int reformat_storage();
```

Comment:
Looks like a public API breakage. Is this something you really want to do?

Reply:
It was only called internally as far as I know.

Reply:
@dhwalters423
Have a look and comment. I understand you need this for automated tests

Reply:
It's needed for the tests - to force the reformatting of the storage so we can test writing of RoT and credentials. Otherwise the filesystem isn't exposed so this is impossible.

Reply:
@dhwalters423 But you pass in the file system in the constructor? So you have access to it from your tests.